### DISCUSSION TITLE
PP-OCRv4_mul

### DISCUSSION THREAD

**Discussion by SeaSpring17** [PaddlePaddle/PaddleOCR#12843] [Q&A]
在PP-OCRv4的introduction种，介绍了PP-OCRv4-mul的多语言的识别模型，并有着显著的提升，然而其给出的model_list.md文件中，仍然只有PP-OCRv2的多语言识别模型，并未看到更新的多语言识别模型

PaddleOCR：2.7

**Comment by DiTo97** [comment]
Any updates on this, @tink2123?

It would be critical for us to evaluate the enhanced v4 version on our dataset.
